lib/matplotlib/axes/_base.py

```diff
@@ -2331,6 +2331,23 @@ def add_child_axes(self, ax):
     def add_collection(self, collection, autolim=True):
         """
         Add a `.Collection` to the Axes; return the collection.
+
+        Parameters
+        ----------
+        collection : `.Collection`
+            The collection to add.
+        autolim : bool
+            Whether to update data and view limits.
+
+            .. versionchanged:: 3.11
+
+               This now also updates the view limits, making explicit
+               calls to `~.Axes.autoscale_view` unnecessary.
+
+            As an implementation detail, the value "_datalim_only" is
```

Comment:
Do we have a deprecation flag for this sort of thing? If not, should manually raise a deprecation warning when this is passed in.

Reply:
Ah understood, I was thinking this was prior behavior for some reason